public/static/partials/user_host_details.html

```diff
@@ -61,5 +61,31 @@ <h2 style="display:inline-block; padding-right:15px">
             </md-button>
         </div>
       </section>
+      <section layout="row" flex ng-show="curHostData.host_type == 'ec2-ondemand' && curHostData.instance_type != ''" style="white-space: nowrap">
+        <div flex="85">
+          <strong>Instance Type:</strong>
+          <span class="dropdown">
+            <button class="btn btn-link btn-dropdown" data-toggle="dropdown" href="#" id="distro">
+              <strong>
+                [[selectedInstanceType]]
+                <span class="fa fa-caret-down"></span>
+              </strong>
+          </button>
+          <ul class="dropdown-menu" role="menu" aria-labelledby="distro">
+            <li role="presentation" class="dropdown-header">Instance types</li>
+            <li role="presentation" ng-repeat="instanceType in allowedInstanceTypes">
+              <a role="menuitem" ng-click="setInstanceType(instanceType)">
```

Comment:
Instead of calling a function, I think you can simplify this by binding the dropdown to $scope.selectedInstanceType

Reply:
Offline: I copied over the distro dropdown menu and don't think I can get it to work with ng-model since it doesn't have a "value" like a select element would -- could make it into a select element but going to leave it for consistency and it works fine I think.